var/spack/repos/builtin/packages/sina/package.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack import *
+
+
+class Sina(CachedCMakePackage):
+    """Sina C++ Library"""
+
+    homepage = 'https://github.com/LLNL/Sina'
+    url = 'https://github.com/LLNL/Sina/releases/download/v1.10.0/sina-cpp-1.10.0.tar.gz'
+
+    maintainers = [
+        'pauli2@llnl.gov',
+        'haluska2@llnl.gov',
+        'murray55@llnl.gov',
+    ]
```

Comment:
These should be GitHub usernames. Spackbot will ping these people when updates to the package are submitted in PRs.

```suggestion
    maintainers = ["estebanpauli", "HaluskaR"]
```

Reply:
I'm not sure what murray55's GitHub username is or if he has one so if you like this you can just commit the suggestion; if not push and we'll get this merged.